src/__test__/Rockets.test.js

```diff
@@ -1,6 +1,6 @@
 import {
   rocketsReducer, getRocketsAction, reserveRocketAction, cancelRocketAction,
-} from './Rockets';
+} from '../Redux/Rockets/Rockets';
 
```

Comment:
Please ensure that you fix this error received when I tried to run the `test`. Kindly fix the error and ensure that all tests are passed.

![image](https://user-images.githubusercontent.com/10332499/159093132-b4637250-a78f-425a-864d-2b29b687fce5.png)

Reply:
Hi @micheaol 

Thank you for your feedback. However, it would be nice you check your system for the testing part because on my end, there was no such error and all tests passed as shown in the image attached.

![Capture](https://user-images.githubusercontent.com/48631109/159094625-ff831b28-8a12-4ba9-91f6-5ab60f467afc.PNG)

You might also want to check out this link for possible fixes: https://github.com/facebook/create-react-app/issues/11792

Thank you.